assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -12,117 +12,117 @@
-# operations that require asynchronous execution e.g. starting/stopping.
-# Possible values are 'fixed', 'cached'
+# Workspace threads pool configuration. This pool is used for workspace-related
+# operations that require asynchronous execution, for example, starting and stopping.
+# Possible values are `fixed` and `cached`.
 che.workspace.pool.type=fixed
 
-# This property is ignored when pool type is different from 'fixed'.
-# Configures the exact size of the pool, if it's set multiplier property is ignored.
-# If this property is not set(0, < 0, NULL) then pool sized to number of cores,
-# it can be modified within multiplier
+# This property is ignored when pool type is different from `fixed`.
+# It configures the exact size of the pool. When set, the `multiplier` property is ignored.
+# If this property is not set (`0`, `<0`, `NULL`), then the pool size equals the number of cores.
+# It can be modified by the `multiplier`.
 che.workspace.pool.exact_size=30
 
-# This property is ignored when pool type is different from 'fixed' or exact pool size is set.
-# If it's set the pool size will be N_CORES * multiplier
+# This property is ignored when pool type is different from `fixed` or an exact pool size is set.
+# When set, the pool size is `N_CORES * multiplier`.
 che.workspace.pool.cores_multiplier=2
 
-# This property specifies how much threads to use for workspaces servers liveness probes
+# This property specifies how many threads to use for workspace server liveness probes.
 che.workspace.probe_pool_size=10
 
 
-# Http proxy setting for workspace JVM
+# HTTP proxy setting for workspace JVM.
 che.workspace.http_proxy_java_options=NULL
 
-# Java command line options to be added to JVM's that running within workspaces.
+# Java command-line options added to JVMs running in workspaces.
 che.workspace.java_options=-XX:MaxRAM=150m -XX:MaxRAMFraction=2 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
 
-# Maven command line options added to JVM's that run agents within workspaces.
+# Maven command-line options added to JVMs running agents in workspaces.
 che.workspace.maven_options=-XX:MaxRAM=150m -XX:MaxRAMFraction=2 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
 
-# Default java command line options to be added to JVM that run maven server.
+# Java command-line options added to the JVM running the Maven server.
 che.workspace.maven_server_java_options=-XX:MaxRAM=128m -XX:MaxRAMFraction=1 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
```

Comment:
I'm confused. What Maven server are we talking about here? We don't have a nexus instance inside Che, and afaiu the only Maven we include is commandline for building projects. The Che server is built with Maven but this sounds like configuration for something else. @skabashnyuk @sleshchenko can you clarify what this does?

Reply:
it's about che6 maven support implementation that is running in che6 workspace